connection.js

```diff
@@ -505,10 +500,10 @@ class Connection {
             .catch(conn_log)
         }
     }
-    processMove(gamedata) {
-        const game = this.connectToGame(gamedata.id)
-        game.makeMove(gamedata.move_number);
-    }
+    // processMove(gamedata) {
+    //     const game = this.connectToGame(gamedata.id)
+    //     game.makeMove(gamedata.move_number);
+    // }
     processStoneRemoval(gamedata) {
```

Comment:
unless i'm mistaken you can also delete or comment out processStoneRemoval

Reply:
on_friendRequest also seems unused:
https://github.com/Dorus/gtp2ogs/blob/CountActiveGames/connection.js#L260-L266

Reply:
Indeed, so many unused functions here. Same with game started method. And delete too if i add it to the ignore list on top.

I think i'm going to make another pr for delete notifications and add this there.

Reply:
yes, making a separate pr would be better, here we're just fixing the connectedgames problem

Reply:
you may want to put back these lines as they were, as i made a specific pr #266